source/nodejs/adaptivecards/src/shared.ts

```diff
@@ -14,27 +25,27 @@ export interface ISeparationDefinition {
 }
 
 export interface IInput {
-	id: string;
-    value: string;
+	id?: string;
+    value?: string;
     validateValue(): boolean;
 }
 
 export type Dictionary<T> = { [key: string]: T };
 
 export class StringWithSubstitutions {
 	private _isProcessed: boolean = false;
-	private _original: string = null;
-    private _processed: string = null;
+	private _original?: string;
+    private _processed?: string;
```

Comment:
mixed tabs and spaces in this file

Reply:
Will fix

Reply:
Addressed with https://github.com/microsoft/AdaptiveCards/pull/3508/commits/34f9d1bc0147914f4dedf54a1367f3ab2e08e4b7